common/app/routes/Hikes/components/Hike.jsx

```diff
@@ -10,7 +10,7 @@ export default contain(
     actions: ['hikesActions']
   },
   React.createClass({
-    displayName: 'Hike',
+    displayName: 'Video',
 
```

Comment:
This PR should not change the internal names of components, just the user facing names.